packages/node/src/handlers/test-api.ts

```diff
@@ -9,17 +9,25 @@ import { WORKER_CALL_API_TIMEOUT } from '../constants';
 
 export async function testApi(
   config: Config,
-  endpointName: string,
+  endpointId: string,
   parameters: Record<string, string>
 ): Promise<[Error, null] | [null, ApiCallResponse]> {
   const testCallId = randomString(8);
   const masterHDNode = wallet.getMasterHDNode();
 
   const logOptions = logger.buildBaseOptions(config, { requestId: testCallId });
 
-  const triggerRequest = find(config.triggers.request, ['endpointName', endpointName]);
+  const triggerRequest = find(config.triggers.request, ['endpointId', endpointId]);
   if (!triggerRequest) {
-    return [new Error(`No such endpoint '${endpointName}'`), null];
+    return [new Error(`No such endpoint with ID '${endpointId}'`), null];
+  }
+
+  const endpoint = find(find(config.ois, ['title', triggerRequest.oisTitle])?.endpoints, [
```

Comment:
```suggestion
  const endpoint = find(config.ois, ['title', triggerRequest.oisTitle, "endpoints", "name", triggerRequest.endpointName])
```

doesn't work?

Reply:
Sry, I was confused. It won't work. But you can make the whole thing more readble by:
```
const endpoints = find(config.ois, ['title', triggerRequest.oisTitle])?.endpoints
const endpoint = find(endpoint, ["name", triggerRequest.endpointName])
```